chapters/annotations.tex

```diff
@@ -1373,6 +1373,8 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 An empty \lstinline!group! (the default) means tool-specific choice of group.
 The order of components (and class definitions) within each group and the order of the groups and tabs are according to the declaration order, where inherited elements are added at the place of the extends.
 
+A variable shall have at most one of \lstinline!showStartAttribute=true!, \lstinline!colorSelector=true!, \lstinline!loadSelector!, \lstinline!saveSelector! or \lstinline!connectorSizing=true!.
```

Comment:
Shouldn't we say _component_ rather than _variable_?

Reply:
Yes, updating.